_includes/api/en/5x/req-host.md

```diff
@@ -0,0 +1,18 @@
+<h3 id='req.host'>req.host</h3>
+
+Contains the hostname and port number derived from the `Host` HTTP header.
```

Comment:
I would suggest this should say _either_ "Contains the host derived [..]" or "Contains the hostname and port number, concatenated with a colon, derived from [...]"

The only downside to the latter, is that it would also need to specify that IPv6 addresses will be wrapped in square brackets as well, since hostname for IPv6 removes the square brackets.

Reply:
@dougwilson  - pushed a change to that effect.
 - rephrased based on your second preference
 - added an IPv6 example for more clarity